src/SignService/Startup.cs

```diff
@@ -91,22 +92,35 @@ public void ConfigureServices(IServiceCollection services)
             JwtSecurityTokenHandler.DefaultInboundClaimTypeMap.Clear();
 
             // Add framework services.
-            services.AddAuthentication(AzureADDefaults.AuthenticationScheme)
-                    .AddAzureAD(options => Configuration.Bind("AzureAd", options))
-                    .AddAzureADBearer(options => Configuration.Bind("AzureAd", options));
+            services.AddAuthentication(OpenIdConnectDefaults.AuthenticationScheme)
+              .AddMicrosoftIdentityWebApp(Configuration)
+                  .EnableTokenAcquisitionToCallDownstreamApi(new[] { "https://graph.windows.net/.default" })
+                     // .AddMicrosoftGraph(Configuration.GetSection("DownstreamApi"))
```

Comment:
We need graph to operate both as the app and as the user based on the context. We basically just call the graph directly as it's 
easier:https://github.com/dotnet/SignService/blob/924022b27795340a93a40f53c37dd80dc6ef0133/src/SignService/Services/IGraphHttpService.cs
https://github.com/dotnet/SignService/blob/924022b27795340a93a40f53c37dd80dc6ef0133/src/SignService/Services/GraphHttpService.cs

Happy to use GraphClient if it can meet our needs and doesn't make things more complicated :)